k8s/namespaces/dm-store/dm-store/perftest.yaml

```diff
@@ -2,14 +2,10 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: dm-store
-#  annotations:
-#    fluxcd.io/automated: "true"
-#    fluxcd.io/tag.java: glob:pr-1229-*
-#    hmcts.github.com/prod-automated: disabled
 spec:
   values:
     java:
-#      image: hmctspublic.azurecr.io/dm/store:pr-1229-067155e-20220119190152
+      image: hmctspublic.azurecr.io/dm/store:prod-d9e0896-20220317083032 #{"$imagepolicy": "flux-system:perftest-dm-store"}
```

Comment:
Still using perftest in automation comment

Reply:
Yes. For now. Perftest will still have same Prod image as other environments. Later in the week, I will amend it to point to PR image.